src/datamigration/azext_datamigration/manual/custom.py

```diff
@@ -281,12 +281,14 @@ def datamigration_login_migration(src_sql_connection_str=None,
                 elif param.__contains__("list") and parameterList[param] is not None:
                     parameterList[param] = " ".join(f"\"{i}\"" for i in parameterList[param])
                     cmd += f' {param} {parameterList[param]}'
+            print("Starting execution...")
```

Comment:
Please leverage `logger.warning` so that this will output to stderr output stream and won't break customer's script for stdout stream

Reply:
These are the info logs that need to be shown to the customer, so logger.warning would not work.